templates/config.edn

```diff
@@ -207,145 +242,179 @@
     :group-by-page? false
     :collapsed? false}]}
 
- ;; Add your own commands to slash menu to speedup.
- ;; E.g.
+ ;; Add custom commands to the command palette
+ ;; Example usage:
  ;; :commands
  ;; [
- ;; ["js" "Javascript"]
- ;; ["md" "Markdown"]
- ;; ]
- :commands
- []
-
- ;; By default, a block can only be collapsed if it has some children.
- ;; `:outliner/block-title-collapse-enabled? true` enables a block with a title
- ;; (multiple lines) can be collapsed too. For example:
+ ;;  ["js" "Javascript"]
+ ;;  ["md" "Markdown"]
+ ;  ]
+ :commands []
+
+ ;; Enable collapsing blocks with titles but no children.
+ ;; By default, only blocks with children can be collapsed.
+ ;; Setting `:outliner/block-title-collapse-enabled?` to true allows collapsing
+ ;; blocks with titles (multiple lines) and content. For example:
  ;; - block title
  ;;   block content
+ ;; Default value: false
  :outliner/block-title-collapse-enabled? false
```

Comment:
default value is nil or false
```cljs
    (let [title-collapse-enabled? (:outliner/block-title-collapse-enabled? (state/get-config))]
      (when (and (not block-ref-with-title?)
                 (seq body)
                 (or (not title-collapse-enabled?)
                     (and title-collapse-enabled?
                          (or (not collapsed?)
                              (some? (mldoc/extract-first-query-from-ast body))))))
        [:div.block-body
         ;; TODO: consistent id instead of the idx (since it could be changed later)
         (let [body (block/trim-break-lines! (:block/body block))]
           (for [[idx child] (medley/indexed body)]
             (when-let [block (markup-element-cp config child)]
               (rum/with-key (block-child block)
```